leap/rk/__init__.py

```diff
@@ -795,4 +801,157 @@ def generate(self):
 # }}}
 
 
+# {{{ Explicit SSP Runge-Kutta methods
+
+class SSPRKMethodBuilder(MethodBuilder):
+    r"""Explicit Strong Stability Preserving (SSP) Runge-Kutta Methods.
+
+    The methods are given in the now-standard Shu-Oscher form
```

Comment:
> not generalized

:shrug: Beats "not existent".

> benefit to writing them in the Osher-Shu form like this.

That's how they appear in the literature, which makes it easy to compare code and literature. That's a good reason to me.

Reply:
Definitely good enough for me too!